lib/turbo_tests/cli.rb

```diff
@@ -101,11 +106,15 @@ def run
         verbose: verbose,
         fail_fast: fail_fast,
         count: count,
+        test_options: test_options
       )
 
       if success
+        STDERR.puts 'success'
         exit 0
       else
+        STDERR.
```

Comment:
Left for debugging?

Reply:
heya thanks for the review, definitely lotsa debugging noise in here.  I'll clean that up and ping you when this is closer to ready for review.